src/components/GameOverScoreboard.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+
+export function GameOverScoreboard({ game }) {
+  let message;
+  let first;
+  let second;
+  let third;
+  let fourth;
+  let fifth;
+  let information;
+
+  function scoreboard() {
+    first = `${game.globalScoreBoard.topScores.first.name}:
+    ${game.globalScoreBoard.topScores.first.score}`;
+    second = `${game.globalScoreBoard.topScores.second.name}:
+    ${game.globalScoreBoard.topScores.second.score}`;
+    third = `${game.globalScoreBoard.topScores.third.name}:
+    ${game.globalScoreBoard.topScores.third.score}`;
+    fourth = `${game.globalScoreBoard.topScores.fourth.name}:
+    ${game.globalScoreBoard.topScores.fourth.score}`;
+    fifth = `${game.globalScoreBoard.topScores.fifth.name}:
+    ${game.globalScoreBoard.topScores.fifth.score}`;
+  }
+
```

Comment:
You can iterate through an object using `Object.keys`. Allowing you to have an array of the high scores. Even if out of order you could simply sort
- `Object.keys`: Converts all of the Object's keys into an array (so you can then just iterate through the keys and pull out the values of the hash)

Using this methodology you could also refactor scoreboard.js